docs/user-docs/experience-blocks-analytics/README.md

```diff
@@ -0,0 +1,38 @@
+# Experience Blocks (XBs)
+
+Altis Experience Blocks (XBs) allow you to create personalized content based on specific user profiles visiting your site. Additionally, XBs can provide insights into the actual performance, analyzing traffic based on the different audiences you've created.
+
+## Creating personalized content
+
+To get started, create a Personalized Content block (aka XB) from the list of available blocks.
+
+![adding experience block](../assets/create-xb.gif)
+
+Add some fallback content. This content will be displayed by default if none of the specific audience variants apply to a user.
+
+Next you'll want to create some personalized content to make the user's experience unique to the audience segment you are trying to reach. You create a variant by clicking the + icon. You will be prompted to select an audience. If no audiences exist, or you want to use a new audience, you can use the "Add New" button to create a new audience. Otherwise you can select an existing audience and add your personalized content for that segment of users.
+
+![Adding an audience](../assets/create-audience-xb.gif)
+
+## Creating audiences
+
+You can create new audiences within the content editing experience or from the Audiences page in the admin. In either case, the process is the same. Within the New Audience screen, you can give your audience a name, and set the variables in the dropdowns to determine how to identify this segment of users, for example browser or geographic region. Once the audience is set, you can use that to deliver unique content based on the parameters you've set. Before you can use the audience, switch the audience to Active on the right hand side. Now the audience is ready to be used in your personalized content!
+
+![Personalized content](../assets/personalized-content.gif)
+
+
+## Add conversion goals to measure engagement
+
+There are three ways that you can track conversions built into Altis Experience Blocks. These are "Impressions", "Click on any link" and "Submit a form". By default, impressions will be selected, and the statistics that display will be based on that data.
+
+![Impressions](../assets/conversions-impressions.png)
+
+However, you can change the conversion goal to any of the available options and the goal tracking will reflect the new goal after the post is updated. Any action of the type selected (clicking a link or submitting a form) will mark a conversion that will be tracked for that variant.
```

Comment:
We should perhaps mention that developers can add additional conversion goals to choose from if the default options don't meet your requirements.